lib/rules/media-feature-range-notation/index.js

```diff
@@ -1,14 +1,20 @@
 'use strict';
 
-const mediaParser = require('postcss-media-query-parser').default;
-
-const { rangeTypeMediaFeatureNames } = require('../../reference/mediaFeatures.js');
 const atRuleParamIndex = require('../../utils/atRuleParamIndex');
-const isRangeContextMediaFeature = require('../../utils/isRangeContextMediaFeature');
-const isStandardSyntaxMediaFeatureName = require('../../utils/isStandardSyntaxMediaFeatureName');
 const report = require('../../utils/report');
 const ruleMessages = require('../../utils/ruleMessages');
 const validateOptions = require('../../utils/validateOptions');
+const parseMediaQuery = require('../../utils/parseMediaQuery');
+const {
+	isMediaFeatureRange,
+	isMediaFeaturePlain,
+	isMediaFeature,
+	MediaFeatureRangeNameValue,
+	MediaFeatureName,
+} = require('@csstools/media-query-list-parser');
+const { rangeTypeMediaFeatureNames } = require('../../reference/mediaFeatures');
+const { TokenNode } = require('@csstools/css-parser-algorithms');
+const { TokenType } = require('@csstools/css-tokenizer');
```

Comment:
[suggest] Can you move the imports of `@csstools/*` to the top like this? In order to distinguish between public packages and internal modules:

```js
'use strict';

const {
	isMediaFeatureRange,
	isMediaFeaturePlain,
	isMediaFeature,
	MediaFeatureRangeNameValue,
	MediaFeatureName,
} = require('@csstools/media-query-list-parser');
const { TokenNode } = require('@csstools/css-parser-algorithms');
const { TokenType } = require('@csstools/css-tokenizer');

const atRuleParamIndex = require('../../utils/atRuleParamIndex');
const { rangeTypeMediaFeatureNames } = require('../../reference/mediaFeatures');
// ...
```

Reply:
Done :)